tests/rbd_mirror/test_rbd_journal_mirroring_with_delay.py

```diff
@@ -0,0 +1,203 @@
+                multi_image_config = getdict(pool_config)
+                multi_image_config.pop("test_config", {})
+                for image, image_config in multi_image_config.items():
+                    image_spec = f"{pool}/{image}"
+                    mount_path = f"/tmp/mnt_{random_string(len=5)}"
+                    rc = run_io_wait_for_replay_complete(
+                        rbd=rbd_obj,
+                        sec_rbd=sec_obj,
+                        client=client_node,
+                        pool=pool,
+                        image=image,
+                        image_config=image_config,
+                        mount_path=f"{mount_path}/file_00",
+                        sec_cluster_name=sec_cluster_name,
+                        skip_mkfs=False,
+                    )
+                    if rc:
+                        log.error(
+                            f"Run IO and wait for replay failed for image {pool}/{image}"
+                        )
+                        return 1
+                    sleep(120)
+                    if delay_at_primary:
+                        rc = config_mirroring_delay(
+                            delay=delay,
+                            delay_per_image=True,
+                            rbd=rbd_obj,
+                            pool=pool,
+                            image=image,
+                            operation="set",
+                        )
+                    else:
+                        rc = config_mirroring_delay(
+                            delay=delay, client=sec_client, operation="set"
+                        )
+                    if rc:
+                        log.error("Setting mirror delay failed")
+                        return 1
+
+                    rc = write_data_and_verify_no_mirror_till_delay(
+                        rbd=rbd_obj,
+                        client=client_node,
+                        sec_rbd=sec_obj,
+                        sec_client=sec_client,
+                        pool=pool,
+                        image=image,
+                        mount_path=f"{mount_path}/file_01",
+                        skip_mkfs=True,
+                        image_config=image_config,
+                        delay=int(delay),
+                    )
+
+                    if rc:
+                        log.error(
+                            f"Verification of no mirror until delay failed for image {image_spec}"
+                        )
+                        return 1
+    except Exception as e:
+        log.error(str(e))
+        return 1
+    finally:
+        if delay_at_primary:
+            rc = config_mirroring_delay(
+                delay_per_image=True,
+                rbd=rbd_obj,
+                pool=pool,
+                image=image,
+                operation="remove",
+            )
+        else:
+            rc = config_mirroring_delay(client=sec_client, operation="remove")
```

Comment:
for secondary, we don't need `pool` and `image` args to remove?

Reply:
we are adding using ceph config at client level, so we're removing as well at client level, its not image or pool level. Please see the config setting code as well above this.